uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Form/Search/StaffDirectory.php

```diff
@@ -445,15 +519,12 @@ private function setQueryConditionForSelectStaff($value) {
     if (in_array($value, array_keys($this->selectStaffFixedOptions))) {
       $jobDetailsCondition = $this->getJobDetailsConditionForSpecificStaff($value);
     }
-    elseif ($value == 'choose_date') {
-      $jobDetailsCondition = $this->getWhereClauseWhenSelectStaffIsChooseDate();
-    }
     else {
-      $jobDetailsCondition = $this->getWhereClauseWhenSelectStaffIsRelativeDate($value);
+      $jobDetailsCondition = $this->getWhereClauseWhenSelectStaffIsChooseDate();
```

Comment:
Guess you wanted to type `getWhereClauseWhenSelectStaffIsChoosenDate` here. Can the name be shortened, probably to `getWhenSelectStaffIsChoosenDate` 🤷‍♂️